corpora/proverbs.dep

```diff
@@ -0,0 +1,1826 @@
+	"ий" v tv pass aor p3 sg @root #2->0
+"<,>"
+	"," cm @punct #3->4
+"<сынбайт>"
+	"сын" v iv neg aor p3 sg @parataxis #4->2
+"<.>"
+	"." sent @punct #5->4
+
+# sent_id = proverbs:71
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:479
+# text = Энеси болушчаактын кызы ыйлаак, атаcы болушчаактын уулу урушчаак.
+# text[eng] = The daughter of a protective mother cries a lot, the son of a protective father gets in fights a lot.
+# labels = JNW_checked_cg tocheck_dep
+"<Энеси>"
+	"эне" n px3sp nom @nmod:poss #1->3
+"<болушчаактын>"
+	"болушчаак" adj subst gen @nmod:poss #2->1
+"<кызы>"
+	"кыз" n px3sp @nsubj #3->4
+"<ыйлаак>"
+	"ыйлаак" adj @root #4->0
+		"э" cop aor p3 sg @cop #5->4
+"<,>"
+	"," cm @punct #6->10
+"<атаcы>"
+	"ата" n px3sp nom @nmod:poss #7->9
+"<болушчаактын>"
+	"болушчаак" adj subst gen @nmod:poss #8->7
+"<уулу>"
+	"уул" n px3sp nom @nsubj #9->10
+"<урушаак>"
+	"урушаак" adj @parataxis #10->4
+		"э" cop aor p3 sg @cop #11->10
+"<.>"
+	"." sent @punct #12->10
+
+# sent_id = proverbs:72
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:482
+# text = Эрди жоо үстүндө көр.
+# text[eng] = Know a man [by what he does] on the battle field.
+# labels = JNW_checked_cg tocheck_dep
+"<Эрди>"
+	"эр" n acc @obj #1->4
+"<жоо>"
+	"жоо" n nom @nsubj #2->4
+"<үстүндө>"
+	"үст" n px3sp loc @obl:lmod #3->1
+"<көр>"
+	"көр" v tv imp p2 sg @root #4->0
+"<.>"
+	"." sent @punct #5->4
+
+# sent_id = proverbs:73
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:401
+# text = Сүйбөгөнгө сүйкөнбө.
+# text[eng] = Don't go after someone who doesn't love you.
+# labels = JNW_checked_cg tocheck_dep
+"<Сүйбөгөнгө>"
+	"сүй" v tv neg gpr_past subst dat @obl #1->2
+"<сүйкөнбө>"
+	"сүйкөн" v iv neg imp p2 sg @root #2->0
+"<.>"
+	"." sent @punct #3->2
+
+# sent_id = proverbs:74
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:232
+# text = Жүктүн оорун нар көтөрөт.
+# text[eng] = The camel carries the weight of the load.
+# labels = JNW_checked_cg tocheck_dep
+"<Жүктүн>"
+	"жүк" n gen @nmod:poss #1->2
+"<оорун>"
+	"оор" adj subst px3sp acc @obj #2->4
+"<нар>"
+	"нар" n nom @nsubj #3->4
+"<көтөрөт>"
+	"көтөр" v tv aor p3 sg @root #4->0
+"<.>"
+	"." sent @punct #5->4
+
+# sent_id = proverbs:75
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:177
+# text = Жалкоого шылтоо көп.
+# text[eng] = A lazy person has lots of excuses.
+# labels = JNW_checked_cg tocheck_dep
+"<Жалкоого>"
+	"жалкоо" n dat @obl #1->2
+"<шылтоо>"
+	"шылтоо" n nom @nsubj #2->3
+"<көп>"
+	"көп" adj @root #3->0
+		"э" cop aor p3 sg @cop #4->3
+"<.>"
+	"." sent @punct #5->4
+
+# sent_id = proverbs:76
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:49
+# text = Алыстагы душмандан аңдып жүргөн дос жаман.
+# text[eng] = A friend who spies on you is worse than an enemy who's far away. 
+# labels = JNW_checked_cg tocheck_dep
+"<Алыстагы>"
+	"алыс" adj subst loc attr @obl:lmod #1->2
```

Comment:
This can't be `@obl` because it's dependent on a noun, not a verb.  But `:lmod` is interesting.  For now, I would just make this `@nmod`.

Reply:
`@nmod:lmod`